src/commons/navigationBar/NavigationBar.tsx

```diff
@@ -75,7 +76,17 @@ const NavigationBar: React.FC = () => {
         key={navbarEntry.text}
       >
         <Icon icon={navbarEntry.icon} />
-        <div>{navbarEntry.text}</div>
+        <div>
+          {
+            <Translation ns={'commons'} keyPrefix={'navigationBar'}>
+              {
+                // @ts-expect-error the t function expects a specific set of strings
+                // as valid translation keys but we are passing in a string type
+                t => t(navbarEntry.text)
```

Comment:
Do you know if it's possible to use a typecast `as SomeType` instead of ignoring the compile error here?

Reply:
we could do something like this:

                t => t(navbarEntry.text as keyof CustomTypeOptions['resources']['commons']['navigationBar'])